src/modules/household/components/householdActions/JoinHouseholdDialog.tsx

```diff
@@ -0,0 +1,277 @@
+import { MergeTypeRounded } from '@mui/icons-material';
+import { Typography } from '@mui/material';
+import React, { useCallback, useMemo, useState } from 'react';
+import Loading from '@/components/elements/Loading';
+import StepDialog, { TabDefinition } from '@/components/elements/StepDialog';
+import {
+  clientBriefName,
+  findHohOrRep,
+  sortHouseholdMembers,
+  stringifyHousehold,
+} from '@/modules/hmis/hmisUtil';
+import AddRelationshipsStep from '@/modules/household/components/householdActions/AddRelationshipsStep';
+import JoinHouseholdReview from '@/modules/household/components/householdActions/JoinHouseholdReview';
+import JoinHouseholdSelectClients from '@/modules/household/components/householdActions/JoinHouseholdSelectClients';
+import SuccessWayfindingStep from '@/modules/household/components/householdActions/SuccessWayfindingStep';
+import {
+  HouseholdClientFieldsFragment,
+  HouseholdFieldsFragment,
+  ProjectAllFieldsFragment,
+  RelationshipToHoH,
+  useGetEnrollmentWithHouseholdQuery,
+  useJoinHouseholdMutation,
+} from '@/types/gqlTypes';
+
+interface Props {
+  open: boolean;
+  conflictingEnrollmentId: string;
+  onClose: VoidFunction;
+  onComplete?: (
```

Comment:
`onSuccess` would be more in line with naming conventions for props elsewhere in the application. This clarifies that the callback is only called if the mutation is successful. (Unlike apollo `onCompleted`)